lib/stream/xlsx/workbook-reader.js

```diff
@@ -25,18 +23,20 @@ class WorkbookReader extends EventEmitter {
     this.options = {
       worksheets: 'emit',
       sharedStrings: 'cache',
-      hyperlinks: 'ignore',
-      styles: 'ignore',
-      entries: 'ignore',
+      hyperlinks: 'cache',
+      styles: 'cache',
+      entries: 'emit',
       ...options,
     };
 
     this.styles = new StyleManager();
     this.styles.init();
+    this._parsedStyles = false;
+    this.waitingWorkSheets = [];
```

Comment:
As far as I understood well, It might be a `const` living in the scope of `parse`. Then passed into `queueWorksheets` and `flushQueue`as a parameter. 
I think that will be much safer, and much more hermetic.

Reply:
Addressed this above so going to mark this as resolved.